base/shell/explorer/syspager.cpp

```diff
@@ -1256,6 +1256,11 @@ void CNotifyToolbar::Initialize(HWND hWndParent, CBalloonQueue * queue)
     tbm.cyBarPad = 1;
     tbm.cxButtonSpacing = 1;
     tbm.cyButtonSpacing = 1;
+    if (!g_TaskbarSettings.bSmallIcons)
```

Comment:
Might be moved just below the original cx/yPad setting done few lines above (1253)